vendordeps/navx_frc.json

```diff
@@ -0,0 +1,35 @@
+{
+    "fileName": "navx_frc.json",
+    "name": "KauaiLabs_navX_FRC",
+    "version": "4.0.447",
+    "uuid": "cb311d09-36e9-4143-a032-55bb2b94443b",
+    "mavenUrls": [
+        "https://repo1.maven.org/maven2/"
+    ],
+    "jsonUrl": "https://www.kauailabs.com/dist/frc/2022/navx_frc.json",
```

Comment:
This is the 2022 version of the NavX library, which is incompatible with the latest WPILib. The 2024 library is already included under vendordeps.